sdk/communication/azure-communication-phonenumbers/test/recordings/test_sip_routing_client_e2e.test_add_trunk.yaml

```diff
@@ -15,31 +15,31 @@ interactions:
       User-Agent:
       - azsdk-python-communication-phonenumbers/1.1.0b3 Python/3.10.0 (Windows-10-10.0.22621-SP0)
       x-ms-date:
-      - Tue, 15 Nov 2022 16:03:45 GMT
+      - Tue, 06 Dec 2022 10:31:17 GMT
       x-ms-return-client-request-id:
       - 'true'
     method: PATCH
     uri: https://sanitized.communication.azure.com/sip?api-version=2021-05-01-preview
   response:
     body:
-      string: '{"trunks":{"sbs1.sanitized.com":{"sipSignalingPort":1122},"sbs2.sanitized.com":{"sipSignalingPort":1123}},"routes":[]}'
+      string: '{"trunks":{"sbs3.3428b2faca6e4813885f4a3708457ed2.com":{"sipSignalingPort":2222}},"routes":[]}'
```

Comment:
It appears that this was not properly sanitized.

Reply:
Fixed sanitizer to cover these cases as well.